front/app/components/admin/PostManager/components/PostTable/Row/InitiativeRow.tsx

```diff
@@ -16,11 +16,14 @@ import {
 import { IInitiativeStatusData } from 'services/initiativeStatuses';
 
 // components
-import { Table, Icon } from 'semantic-ui-react';
-import WrappedRow from './WrappedRow';
+import { TitleLink } from '.';
+import StyledRow from './StyledRow';
+import { Td } from 'components/admin/Table';
+import { Icon } from 'semantic-ui-react';
```

Comment:
Definitely, but they're still used all over the place unfortunately. Would be better to do them in one go in collaboration with the design team, so that they can pick which `semantic-ui` icons we replace with our own icons etc.